addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -155,13 +157,13 @@
                     <field name="remaining_hours_available" invisible="1"/>
                     <span id="remaining_hours_so_label" attrs="{'invisible': ['|', '|', '|', '|', ('allow_billable', '=', False), ('sale_order_id', '=', False), ('partner_id', '=', False), ('sale_line_id', '=', False), ('remaining_hours_available', '=', False)]}">
                         <label class="font-weight-bold" for="remaining_hours_so" string="Remaining Hours on SO"
-                               attrs="{'invisible': ['|', ('encode_uom_in_days', '=', True), ('remaining_hours_so', '&lt;', 0)]}"/>
+                                attrs="{'invisible': ['|', ('encode_uom_in_days', '=', True), ('remaining_hours_so', '&lt;', 0)]}"/>
                         <label class="font-weight-bold" for="remaining_hours_so" string="Remaining Days on SO"
-                               attrs="{'invisible': ['|', ('encode_uom_in_days', '=', False), ('remaining_hours_so', '&lt;', 0)]}"/>
+                                attrs="{'invisible': ['|', ('encode_uom_in_days', '=', False), ('remaining_hours_so', '&lt;', 0)]}"/>
                         <label class="font-weight-bold text-danger" for="remaining_hours_so" string="Remaining Hours on SO"
-                               attrs="{'invisible': ['|', ('encode_uom_in_days', '=', True), ('remaining_hours_so', '&gt;=', 0)]}"/>
+                                attrs="{'invisible': ['|', ('encode_uom_in_days', '=', True), ('remaining_hours_so', '&gt;=', 0)]}"/>
                         <label class="font-weight-bold text-danger" for="remaining_hours_so" string="Remaining Days on SO"
-                               attrs="{'invisible': ['|', ('encode_uom_in_days', '=', False), ('remaining_hours_so', '&gt;=', 0)]}"/>
+                                attrs="{'invisible': ['|', ('encode_uom_in_days', '=', False), ('remaining_hours_so', '&gt;=', 0)]}"/>
```

Comment:
Why do you add a space ? 🤔 
IMHO, it is useless to add a space and don't reformat the code that you don't update because we lose the main commit for a real update in these lines.